pkgs/os-specific/linux/wooting-udev-rules/default.nix

```diff
@@ -2,9 +2,9 @@
 
 stdenv.mkDerivation rec {
   pname = "wooting-udev-rules";
-  version = "20210525";
+  version = "20230331";
```

Comment:
```suggestion
  version = "unstable-2023-03-31";
```

Reply:
Putting "unstable" in there kinda implies there is a stable version, no?

But it's fine. I changed it.

Reply:
unstable means there is no tagged version upstream in nixpkgs